app/assets/stylesheets/dance_comments.scss

```diff
@@ -0,0 +1,22 @@
+#comment_box {
+  padding: 20px;
+  margin: 5px;
+  border: 3px solid black;
+  position: relative;
+}
+
+#comment_box p {
+  padding-left: 5px;
+}
+
+#comment_box footer ul {
```

Comment:
`Selector should have depth of applicability no greater than 2, but was 3`

This rule kind of makes sense. I think a good compromise here is to use scss nested selectors:

```
#comment_box {
  // styles
  p {
    // styles
  }
  
  footer ul {
    // styles
  }
}
```